common/SmokeTests/SmokeTest/test-resources.json

```diff
@@ -53,6 +54,25 @@
                 "kafkaEnabled": false
             }
         },
+        {
+            "type": "Microsoft.Devices/IotHubs",
+            "apiVersion": "2020-01-01",
```

Comment:
This was listed as supported in the virginia govcloud instance.   Will give it a run to see....

Reply:
I assume this sample already has a arm template for the resources can we instead make the smoke tests aggregate each of those arm-templates instead of duplicating them here?

Reply:
Good thought.   I changed this over to a linked template.